src/pages/ChooseArea.svelte

```diff
@@ -142,40 +151,52 @@
   }
 </script>
 
-<div class="left">
-  <h1>Welcome to ATIP v2</h1>
-  <button type="button" on:click={() => (showAbout = !showAbout)}>About</button>
-
-  <p>Select Transport Authority or Local Authority District:</p>
-  <div>
-    <input
-      data-testid="transport-authority"
-      list="authorities-list"
-      bind:value={inputValue}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half left">
+    <h1 class="govuk-heading-l">Welcome to ATIP v2</h1>
+    <SecondaryButton on:click={() => (showAbout = !showAbout)}
+      >About</SecondaryButton
+    >
+
+    <FormElement
+      label="Select Transport Authority or Local Authority District"
+      id="inputValue"
+    >
+      <input
+        class="govuk-input govuk-input--width-20"
```

Comment:
Here's a case where it's simplest to just directly use govuk classes, rather than refactor a component yet. I couldn't find any guidance on text input fields with custom autocomplete like this.